text/0199-support-elastic-common-schema-in-opentelemetry.md

```diff
@@ -5,8 +5,9 @@
 This proposal is to merge the Elastic Common Schema (ECS) with the OpenTelemetry Semantic Conventions (SemConv) and provide full interoperability in OpenTelemetry component implementations. We propose to implement this by aligning the OpenTelemetry Semantic Conventions with [ECS FieldSets](https://www.elastic.co/guide/en/ecs/current/ecs-field-reference.html#ecs-fieldsets) and vice versa where feasible. The long-term goal is to achieve convergence of ECS and OTel Semantic Conventions into a single open schema (e.g. OpenTelemetry Common Schema).
 
 ## The Goal
-- Long-term, ECS and OTel SemConv will converge into one open standard that is maintained by OpenTelemetry (with Elastic contributing, participating and co-maintaining the new standard). 
-- OTel SemConv will adopt ECS in its full scope (except for individual adjustments in detail where needed), including the logging, observability and security domain fields, to make the new schema a true successor of ECS and OTel SemConv.
+- Long-term, ECS and OTel SemConv will converge into one open standard that is maintained by OpenTelemetry. To kick off this effort, Elastic will nominate several domain experts to join the OpenTelemetry Semantic Convention Approvers to help with maintaining the new standard.
```

Comment:
I think "several" is good enough and we shouldn't put specific numbers (e.g. 3) in the OTEP because OTEP is capturing the direction rather than providing all the details.

Regarding identifying the OpenTelemetry Semantic Convention Approvers, @jsuereth is on point and I'm helping, I think the TC members will make proper decision similar to what we did for previous donations.

Reply:
ok. i don't have permission to resolve my own conversations for some reason